### PR TITLE
Migrate to amo_urdf from odio_urdf

### PR DESCRIPTION
**Pull request by akhilsathuluri**
Have updated the `DrakeSimulator` class and tested it with the `amo_urdf`, can confirm the components working

### CODE REVIEW

File: examples/drake_walking.ipynb [hunk 1]
```diff
@@ -250,7 +250,7 @@
     "            break\n",
     "\n",
     "    # Reading new references\n",
-    "    com, dcom, forces_left, forces_right = mpc.get_references()\n",
+    "    com, dcom, forces_left, forces_right, ang_mom = mpc.get_references()\n",
```
Comment (CarlottaSartore) [inline examples/drake_walking.ipynb:253]: This seems not related to the change of the PR, is it a leftover ?

**Review by CarlottaSartore — CHANGES_REQUESTED** [review]
Thank you @akhilsathuluri  ! I have a small comment!

File: examples/drake_walking.ipynb (hunk 1 above)
Comment (akhilsathuluri) [reply examples/drake_walking.ipynb:253]: Yes, when I pulled the latest version of comodo, I got a `too many to unpack` error. I saw the `mujoco` to see that this was added. So corrected it. You are right, it is not relevant to the migration to `amo_urdf`

**Review by CarlottaSartore — APPROVED** [review]

**Comment by CarlottaSartore** [comment]
Looks good ! Merging !
